tests/cypress/support/e2e.ts

```diff
@@ -0,0 +1,10 @@
+import './commands/global'
+import './triggers/before'
```

Comment:
I'd love to see absolute imports here, today's common practice is to use `@/...whatever` (see the comment in tsconfig)